cheroot/errors.py

```diff
@@ -46,8 +46,9 @@ def plat_specific_errors(*errnames):
 )
 socket_errors_to_ignore.append('timed out')
 socket_errors_to_ignore.append('The read operation timed out')
-if sys.platform == 'darwin':
-    socket_errors_to_ignore.append(plat_specific_errors('EPROTOTYPE'))
-
 socket_errors_nonblocking = plat_specific_errors(
     'EAGAIN', 'EWOULDBLOCK', 'WSAEWOULDBLOCK')
+
+if sys.platform == 'darwin':
```

Comment:
It's osx specific, just like the previous fix. It's all in the article that's linked from the previous fix: https://github.com/cherrypy/cherrypy/commit/2ad0cb2e211f3ca9ae81b061dd9fa90ed0268f80

Summary: socket isn't ready yet, Apple made there own error code. Only happens from time to time.